tests/test_skew_symmetric_matrices.py

```diff
@@ -68,3 +68,12 @@ def test_basis_representation_is_correctly_vectorized(self):
             shape = gs.shape(skew.basis_representation(skew.basis))
             dim = int(n * (n - 1) / 2)
             self.assertEqual(shape, (dim, dim))
+
+    @geomstats.tests.np_and_pytorch_only
```

Comment:
what is the limitation for tf here?

Reply:
The limitation is in `skew.basis_representation`: https://github.com/geomstats/geomstats/blob/f908c173064773a888dfc61e1510acf1e4546eae/geomstats/geometry/skew_symmetric_matrices.py#L94-L97
This doesn't work with tf because indexing by an array doesn't work like in numpy. The method tf.gather_nd could be used, but it would have to go in the backend..

Reply:
Something simpler and cleaner also works with pytorch and numpy but not tf:
````python
rows, cols = gs.triu_indices(self.n, k=1)
result_ = matrix_representation[..., rows, cols]

Reply:
I created a triu_to_vec function in the back-ends